script/testing/oltpbench/README.md

```diff
@@ -1,18 +1,94 @@
 # OLTP Benchmark Testing
 This folder include the necessary files for running oltpbench testings.
 
```

Comment:
Lets add a "Table of contents" at the top. That should help people know what is on this page since there is a lot of good content

Reply:
done.